themes/psh-docs/layouts/shortcodes/frameworks.md

```diff
@@ -1,7 +1,24 @@
+{{ $version := .Get "version" }}
+{{ if $version }}
+    {{ if eq ( string .Site.Params.vendor.config.version ) $version }}
```

Comment:
so if i understand it well: 
- platform.vendor.config.version=1
- friday.vendor.config.version=2 
?

Reply:
Correct